src/sdk/src/Layout/pkg/dotnet-sdk.proj

```diff
@@ -8,6 +8,7 @@
     <GenerateInstallers Condition="'$(DotNetBuildSourceOnly)' != 'true'">true</GenerateInstallers>
     <BuildDebPackage Condition="'$(BuildSdkDeb)' == 'true'">true</BuildDebPackage>
     <BuildRpmPackage Condition="'$(BuildSdkRpm)' == 'true'">true</BuildRpmPackage>
+    <UseArcadeRpmTooling>true</UseArcadeRpmTooling>
```

Comment:
This is regressing https://github.com/dotnet/dotnet/commit/eaa19c281d34580a8168cff9ce1e7337da8bfe4f#diff-1383c66abf19740458da3c23699cfe5dac330d777c77bffa919fed432158b14e

We've had the same issue in Preview 7. The result was missing SDK package dependencies for both DEB and RPM. @mmitche fixed the backflow/forward-flow issue with https://github.com/dotnet/dotnet/commit/544d4310bec510b8472a773ea3ceb74af499dc7d (main) and the fix is in RC1 branch.

But, this SDK flow is trying to revert the change, again. @premun @marcpopMSFT - can this be change be excluded from the PR and root cause determined so we don't hit this again?

@MichaelSimons - fyi

Reply:
I meanwhile cherry-picked that commit into this branch.